.github/workflows/knownprojects_build.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     name: build
     runs-on: ubuntu-22.04
     container:
-      image: nycplanning/build-base:${{ inputs.image_tag }}
+      image: nycplanning/build-base:${{ inputs.image_tag || 'latest' }}
```

Comment:
How did I do this right for all of them except one?

Thank you!

Reply:
knowing I untangled some merge conflicts in a KPDB branch recently, it's also possible I undid your making this change